### DISCUSSION TITLE
Adding Second Root Folder

### DISCUSSION THREAD

**Discussion by ccigas** [meisnate12/Plex-Meta-Manager#426] [Q&A]
I recently added a second server to my storage for Plex. I added the root folders for this new location in my Radarr and Sonarr but how would I add the second folders to my config.yaml for the Meta Manager?

**Comment by xlTOTALxCHAOSlx** [comment]
bump

**Comment by meisnate12** [comment]
which root folder do you want sent to radarr/sonarr? if you're using the new space for downloads add that. The root_folder options tells PMM what to tell radarr/sonarr when adding items to them. you can also set the root folder per collection and per library if needed but you have to pick at some point where its being stored definitely

**Reply by xlTOTALxCHAOSlx** [reply]
My Plex Servers root folders currently consists of:
"/volume1/media/Movies/"
"/volume1/media/TV Shows/"
"/volume2/archive/Movies/"
"/volume2/archive/TV Shows/"

Can I list 2 root Movie folders in the config for Radarr? Or should I set it per Library?

**Reply by meisnate12** [reply]
why do you want to list two you have to pick where you want new movies added pick the one you want new movies added to

**Reply by kevbentz** [reply]
> My Plex Servers root folders currently consists of: "/volume1/media/Movies/" "/volume1/media/TV Shows/" "/volume2/archive/Movies/" "/volume2/archive/TV Shows/"
> 
> Can I list 2 root Movie folders in the config for Radarr? Or should I set it per Library?

You can have two (or more) Root Folders in Radarr.  Just enter like you did the first root folder and make sure the path points at "/volume2/archive/Movies".   When you add a new movie in Radarr you will have to pick which root folder to add it to (from the drop down) and in PMM you'll have to preselect the root folder in the config file.
